.pipeline/scripts/set-release-versions.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import os
+import re
+import sys
+from xml.etree import ElementTree
+
+class _CommentedTreeBuilder(ElementTree.TreeBuilder):
+    def __init__(self, *args, **kwargs):
+        super(_CommentedTreeBuilder, self).__init__(*args, **kwargs)
+
+    def comment(self, data):
+        self.start(ElementTree.Comment, {})
+        self.data(data)
+        self.end(ElementTree.Comment)
+
+def _update_readme(sdk_version):
+    _update_file("README.md", r'(maven_central-)(.+)(-blue\.svg)', r'\g<1>%s\g<3>' % sdk_version)
+    _update_file("README.md", r'(a:sdk-core%20AND%20v:)(.+\))', r'\g<1>%s)' % sdk_version)
```

Comment:
(Comment)

This looks like it should better be a regular grep/sed command than a python script :)

Reply:
Nevermind, you're just reusing the existing method.

Personally I don't see the benefit of using Python over `grep` / `sed`. But that's not a hill I'm planning to die on :)